### PR TITLE
Allowing for co-existing Axon agents

### LINKED ISSUE #8: No way to override service name
Different agent builds will have different service names, the service name should be overridable.  

This will specifically be a need for the ip360 and tlc "coexisting" agents.

### LINKED ISSUE #9: No way to override the config directory.
Different agent builds will have different conf directories, thus it should be overridable.  

This will specifically be a need for the ip360 and tlc "coexisting" agents.   

Example, the linux ip360 agent's directory is /etc/tripwire-ip360/

### PR DESCRIPTION
**Pull request by effective-spoons**
== STORY / BUGS

Resolves #8
Resolves #9

== DETAILS

Enhancing the cookbook to allow users to set the service name for
the IP360 Axon agent as well as set its configuration directory
path.

Minor fixes to the custom resource to validate co-existing agents
can be installed by a wrapper cookbook.

== TESTING

kitchen test
foodctiric
cookstyle

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -221,6 +221,26 @@ suites:
...
+      - win2k12r2
+      - debian9
+      - ubuntu1604
+    attributes:
+      tripwire_agent:
+        installer: '/path/to/axon-agent-installer-linux-x64.rpm'
+        axon:
+          bridge: 'tw-bridge.example.com'
+          registration_key: '123PAs5W0rD'
+          install_directory: '/opt/tirpwire-ip360'
```
Comment (DavidWhitlock) [inline .kitchen.yml]: Type-o?  `tirpwire`

File: .kitchen.yml [hunk 2]
```diff
@@ -304,6 +324,25 @@ suites:
...
+    excludes:
+      - a_windows_type
+      - a_centos_type
+    attributes:
+      tripwire_agent:
+        installer: '/path/to/axon-agent-installer-linux-x64.deb'
+        axon:
+          bridge: 'tw-bridge.example.com'
+          registration_key: '123PAs5W0rD'
+          install_directory: '/opt/tirpwire-ip360'
```
Comment (DavidWhitlock) [inline .kitchen.yml]: Again, type-o?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
 # tripwire_agent Cookbook CHANGELOG
 
+## 0.1.5 (TBD)
+
+Expanding axon's resources to support co-existing agents for TE, TLC, and IP360
```
Comment (DavidWhitlock) [inline CHANGELOG.md:5]: I'm not sure if TLC is supported in this scenario yet.  You might want to double-check that.

**Review by DavidWhitlock — CHANGES_REQUESTED** [review]
I'll defer to someone else for approval.  I found a couple of  type-os, though.

**Review by DavidWhitlock — COMMENTED** [review]
Thank you for fixing those.

**Review by DavidWhitlock — APPROVED** [review]
Okay.  Let's get these changes merged in.

**Review by hackalyze — APPROVED** [review]

**Review by DavidWhitlock — APPROVED** [review]
Let's try this again.  I still approve.

**Pull request merged**